<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据标注</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="selector">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>330</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>是否能观测到病灶</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layout">
      <item>
       <widget class="QRadioButton" name="yes">
        <property name="text">
         <string>是</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="no">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>否</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="invalid">
        <property name="text">
         <string>无效图片</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">group</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>821</width>
      <height>651</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hint">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_open"/>
   <addaction name="act_first"/>
   <addaction name="act_prev"/>
   <addaction name="act_next"/>
  </widget>
  <action name="act_open">
   <property name="text">
    <string>打开文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="act_first">
   <property name="text">
    <string>首个未标注</string>
   </property>
  </action>
  <action name="act_next">
   <property name="text">
    <string>下个未标注</string>
   </property>
  </action>
  <action name="act_prev">
   <property name="text">
    <string>上个未标注</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="group">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
